src/style/globalStyle.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+html {
+    box-sizing: border-box;
+}
+*, ::after, ::before {
+    box-sizing: inherit;
+}
+body{
+    margin:0;
+    padding:0
+    }
+`;
```

Comment:
1. I think Prettier here doesn't work, it should have spaces
2.  `*` selector should be first and this one should inherit from html, rather box-sizing is usually set for all elements 

about css reset: https://www.joshwcomeau.com/css/custom-css-reset/